search/feature_offset_match.hpp

```diff
@@ -386,10 +378,11 @@ template <typename TFilter, typename ToDo>
 void MatchFeaturesInTrie(SearchQueryParams const & params, trie::DefaultIterator const & trieRoot,
                          TFilter const & filter, ToDo && toDo)
 {
-  TrieValuesHolder<TFilter> categoriesHolder(filter);
+  using TValue = trie::DefaultIterator::TValue;
+  TrieValuesHolder<TFilter, TValue> categoriesHolder(filter);
   bool const categoriesMatched = MatchCategoriesInTrie(params, trieRoot, categoriesHolder);
 
-  impl::OffsetIntersecter<TFilter> intersecter(filter);
+  impl::OffsetIntersecter<TFilter, TValue> intersecter(filter);
```

Comment:
Are you sure that OffsetIntersecter works fine with CBVs? I don't see Intersect() calls...